.github/workflows/kind-e2e.yaml

```diff
@@ -73,6 +79,27 @@ jobs:
         kubectl wait --timeout=60s --for=condition=Ready pod/kodata
         kubectl delete pod kodata
 
+
+    - name: Check SBOM
+      run: |
+        set -o pipefail
+        IMAGE=$(ko publish ./test)
+        SBOM=$(cosign download sbom ${IMAGE})
+        KO_DEPS=$(ko deps ${IMAGE})
+
+        echo '::group:: SBOM'
+        echo "${SBOM}"
+        echo '::endgroup::'
+
+        echo '::group:: ko deps'
+        echo "${KO_DEPS}"
+        echo '::endgroup::'
+
+        if "${SBOM}" != "${KO_DEPS}" ; then
```

Comment:
I actually expect this to fail for now because of differences in the file path, but let's see what we get.

Reply:
It didn't fail, so I'm trying to make it fail, so I can then fix it.  The first lines definitely don't match.

Reply:
Awesome, it fails now: https://github.com/google/ko/runs/4274269864?check_suite_focus=true#step:11:49

I'll fix this when I'm back from breakfast.